openeogeotrellis/backend.py

```diff
@@ -521,6 +521,11 @@ def file_formats(self) -> dict:
                             "type": ["object"],
                             "description": "Sets file-level GDAL metadata (key-value).",
                         },
+                        "bands_metadata": {
+                            "type": ["object"],
```

Comment:
if it's optional (None by default), the type should reflect that:
```suggestion
                            "type": ["object", "null"],
```